app/src/main/java/com/pluu/webtoon/data/daum/DaumWeekApi.kt

```diff
@@ -0,0 +1,97 @@
+package com.pluu.webtoon.data.daum
+
+import com.pluu.kotlin.asSequence
+import com.pluu.kotlin.isNotEmpty
+import com.pluu.webtoon.common.Const
+import com.pluu.webtoon.data.IRequest
+import com.pluu.webtoon.data.REQUEST_METHOD
+import com.pluu.webtoon.data.WeeklyRequest
+import com.pluu.webtoon.data.impl.AbstractWeekApi
+import com.pluu.webtoon.di.NetworkUseCase
+import com.pluu.webtoon.item.Result
+import com.pluu.webtoon.item.Status
+import com.pluu.webtoon.item.ToonInfo
+import com.pluu.webtoon.utils.safeAPi
+import org.json.JSONArray
+import org.json.JSONObject
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/e6d7b5dbb11d2d4fb917c63fbb0187d6361e5aea/app/src/main/java/com/pluu/webtoon/data/daum/DaumWeekApi.kt#L18"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>